### PR TITLE
feat(scheduler-core): time scale should take fractional startDayHour/endDayHour values

### PR DESCRIPTION
**Pull request by churkin**

### CODE REVIEW

**Comment by MaximKudriavtsev** [comment]
Can we update our documentation to inform customers about updates? I mean adding the following sentence `Accepts floating-point numbers from 0 to 24.` Like we've already done [here](https://js.devexpress.com/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/views/#endDayHour).

**Comment by churkin** [comment]
@RomanTsukanov ping

File: packages/dx-react-scheduler/docs/reference/week-view.md [hunk 1]
```diff
@@ -31,8 +31,8 @@ displayName? | string |  | The view's name used in UI plugins. The default value
 excludedDays? | Array&lt;number&gt; | [] | Specifies the days of week that should not be displayed in the view. Accepts an array of zero-bazed day indexes (0 - Sunday).
 intervalCount? | number | 1 | Multiplies the default view interval.
 cellDuration? | number | 30 | Specifies the cell's duration in minutes.
-startDayHour? | number | 0 | Specifies the start hour of the view time scale.
-endDayHour? | number | 24 | Specifies the end hour of the view time scale.
+startDayHour? | number | 0 | Specifies the start hour of the view time scale. Accepts floating values from 0 to 24.
```
Comment (RomanTsukanov) [inline packages/dx-react-scheduler/docs/reference/week-view.md]:
```suggestion
startDayHour? | number | 0 | Specifies the start hour of the view time scale. Accepts floating-point numbers from 0 to 24.
```

File: packages/dx-react-scheduler/docs/reference/week-view.md [hunk 2]
```diff
@@ -31,8 +31,8 @@ displayName? | string |  | The view's name used in UI plugins. The default value
...
+endDayHour? | number | 24 | Specifies the end hour of the view time scale. Accepts floating values from 0 to 24.
```
Comment (RomanTsukanov) [inline packages/dx-react-scheduler/docs/reference/week-view.md]:
```suggestion
endDayHour? | number | 24 | Specifies the end hour of the view time scale. Accepts floating-point numbers from 0 to 24.
```

**Comment by churkin** [comment]
FPR

**Pull request merged**